asyncapi-parser.json

```diff
@@ -0,0 +1,1774 @@
+    },
+    {
+      "file": "/tests/asyncapi-2.0/AsyncAPI Object/Fields Types/invalid-servers-type.yaml",
+      "success": false,
+      "error": "Error: There were errors validating the AsyncAPI document."
+    },
+    {
+      "file": "/tests/asyncapi-2.0/AsyncAPI Object/Fields Types/invalid-tags-type.yaml",
+      "success": false,
+      "error": "Error: There were errors validating the AsyncAPI document."
+    },
+    {
+      "file": "/tests/asyncapi-2.0/AsyncAPI Object/invalid-duplicate-tags.yaml",
+      "success": true
+    },
+    {
+      "file": "/tests/asyncapi-2.0/AsyncAPI Object/invalid-missing-asyncapi.yaml",
+      "success": false,
+      "error": "Error: The `asyncapi` field is missing."
+    },
+    {
+      "file": "/tests/asyncapi-2.0/AsyncAPI Object/invalid-missing-channels.yaml",
+      "success": false,
+      "error": "Error: There were errors validating the AsyncAPI document."
+    },
+    {
+      "file": "/tests/asyncapi-2.0/AsyncAPI Object/invalid-missing-info.yaml",
+      "success": false,
+      "error": "Error: There were errors validating the AsyncAPI document."
+    },
+    {
+      "file": "/tests/asyncapi-2.0/AsyncAPI Object/valid.yaml",
+      "success": true
+    },
+    {
+      "file": "/tests/asyncapi-2.0/AsyncAPI Version String/invalid-missing-patch.yaml",
+      "success": false,
+      "error": "Error: parsedJSON.asyncapi.startsWith is not a function"
+    },
+    {
+      "file": "/tests/asyncapi-2.0/AsyncAPI Version String/invalid-version-string-format.yaml",
+      "success": false,
+      "error": "Error: Version welcome is not supported."
+    },
+    {
+      "file": "/tests/asyncapi-2.0/AsyncAPI Version String/valid-major-minor-patch.yaml",
+      "success": true
+    },
+    {
+      "file": "/tests/asyncapi-2.0/AsyncAPI Version String/valid-with-hyphen.yaml",
+      "success": true
+    },
+    {
+      "file": "/tests/asyncapi-2.0/Channel Bindings Object/amqp/Fields Types/invalid-bindingVersion-type.yaml",
+      "success": true
+    },
+    {
+      "file": "/tests/asyncapi-2.0/Channel Bindings Object/amqp/Fields Types/invalid-exchange-autoDelete-type.yaml",
+      "success": true
+    },
+    {
+      "file": "/tests/asyncapi-2.0/Channel Bindings Object/amqp/Fields Types/invalid-exchange-durable-type.yaml",
+      "success": true
+    },
+    {
+      "file": "/tests/asyncapi-2.0/Channel Bindings Object/amqp/Fields Types/invalid-exchange-name-type.yaml",
+      "success": true
+    },
+    {
+      "file": "/tests/asyncapi-2.0/Channel Bindings Object/amqp/Fields Types/invalid-exchange-type-type.yaml",
+      "success": true
+    },
+    {
+      "file": "/tests/asyncapi-2.0/Channel Bindings Object/amqp/Fields Types/invalid-exchange-type.yaml",
+      "success": true
+    },
+    {
+      "file": "/tests/asyncapi-2.0/Channel Bindings Object/amqp/Fields Types/invalid-exchange-vhost-type.yaml",
+      "success": true
+    },
```

Comment:
some invalid files that passed.

Reply:
yes, it is possible that some invalid files will pass because parser doesn't have 100% coverage yet 😞 like for example we do not validate payload examples yet. Have a look at the report https://asyncapi.github.io/tck/

maybe before we actually can use TCK we should first make sure the parser has 100% coverage for both, valid and invalid files 🤔